package-lock.json

```diff
@@ -14,7 +14,7 @@
         "@angular/common": "~13.3.10",
         "@angular/compiler": "~13.3.10",
         "@angular/core": "~13.3.10",
-        "@angular/elements": "^13.3.10",
+        "@angular/elements": "~13.3.10",
```

Comment:
Why Semantic versioning rules (~, ^, etc) are used in package-lock.json? I thought that the point of lock.json file is really to lock versions. Apologies if there is no any sense in question :-)